OWNERS_ALIASES

```diff
@@ -61,11 +61,7 @@ aliases:
     - MadhavJivrajani
     - Priyankasaggu11929
     - cblecker
-    - jberkus
-    - kaslin
```

Comment:
So historically `-leads` has included both chairs and techleads.
I might suggest splitting out a distinct alias to avoid confusion.

Reply:
Let's keep the `-leads` group as it is, as this might be used at other places in this repo as well. For the specific changes here, we should have a new alias like `sig-contributor-experience-technical-leads` or something like that and then use the same in f9389189b8873a5997d0750e480b92fcaf51aff5ef07c0111ca0bda6a72ffceaR19

Reply:
Done